include/envoy/runtime/runtime.h

```diff
@@ -96,6 +113,12 @@ class Snapshot {
    * @return uint64_t the runtime value or the default value.
    */
   virtual uint64_t getInteger(const std::string& key, uint64_t default_value) const PURE;
+
+  /**
+   * Fetch the raw runtime entries map.
```

Comment:
Can you add something to this comment about the lifespan of the `const Entry*`?

Reply:
Done